prosody/Dockerfile

```diff
@@ -1,10 +1,30 @@
 FROM jitsi/base
 
+ADD https://prosody.im/files/prosody-debian-packages.key /tmp/key
+
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
```

Comment:
Please pin this to a specific commit, so we can control what version of the file is pulled.

Reply:
Done.